organization_membership.go

```diff
@@ -77,6 +77,9 @@ type OrganizationMembershipListOptions struct {
 	// Optional: A list of relations to include. See available resources
 	// https://www.terraform.io/cloud-docs/api-docs/organization-memberships#available-related-resources
 	Include []OrgMembershipIncludeOpt `url:"include,omitempty"`
+
+	// Optional: A list of organization member emails to filter by.
+	Emails []string `url:"filter[email],omitempty"`
```

Comment:
should this be, 
```suggestion
	Emails []string `url:"filter[emails],omitempty"`
```

Reply:
👋 The param in the API is actually `filter[email]`: https://www.terraform.io/cloud-docs/api-docs/organization-memberships#query-parameters